fivetran_provider/operators/fivetran.py

```diff
@@ -44,6 +44,8 @@ class FivetranOperator(BaseOperator):
     :param connector_id: ID of the Fivetran connector to sync, found on the
         Connector settings page.
     :type connector_id: str
+    :param manual: manual schedule flag, disable to keep connector on Fivetran schedule
+    :type manual: bool
```

Comment:
Please document what this does here. e.g.

If True, completely disables the Fivetran-managed schedule. Default False.

Reply:
Whoops, Default True.